frontend/sozisel-app/src/components/SessionsList/SessionCard/SessionCard.tsx

```diff
@@ -0,0 +1,132 @@
+import "./SessionCard.scss";
+
+import { BaseSyntheticEvent, ReactElement, useState } from "react";
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Dialog,
+  DialogContent,
+  IconButton,
+  Typography,
+} from "@material-ui/core";
+
+import CloseIcon from "@material-ui/icons/Close";
+import DeleteIcon from "@material-ui/icons/Delete";
+import FileCopyIcon from "@material-ui/icons/FileCopy";
+import { Session } from "../../../graphql";
+import ShareIcon from "@material-ui/icons/Share";
+import getSessionStatus from "../../../model/utils/session";
+import useAvatarById from "../../../hooks/useAvatarById";
+import { useTranslation } from "react-i18next";
+
+export interface SessionCardProps {
+  key: string;
+  session: Session;
+  onDelete: (id: string) => void;
+}
+
+export default function SessionCard({
+  session,
+  onDelete,
+}: SessionCardProps): ReactElement {
+  const { t } = useTranslation("common");
+  const sessionLink = `https://sozisel.pl/sessions/${session.id}/join`;
```

Comment:
Use window location for that. Right now it will only work on production environment (provided that we get that domain) but when you use window location then it will work on any environment.